src/replication/protocol.rst

```diff
@@ -1644,20 +1636,12 @@ Replicator updates Replication Log on Source:
 
         PUT /target/_local/afa899a9e59589c3d4ce5668e3218aef HTTP/1.1
         Accept: application/json
-        Content-Length: 591
         Content-Type: application/json
         Host: localhost:5984
         User-Agent: CouchDB
 
         {
             "_id": "_local/afa899a9e59589c3d4ce5668e3218aef",
-            "_rev": "1-31f36e40158e717fbe9842e227b389df",
```

Comment:
I would prefer to show here the case when Source and Target are out of sync of checkpoint document revisions. That's fine and could be, but doesn't really matters as like as the local document revisions. CouchDB uses `0-N`, some else compatible service may provide own ones. That's not a strict rule to follow.